CHANGELOG.md

```diff
@@ -15,7 +15,7 @@ SPDX-License-Identifier: CC-BY-SA-4.0
     - `{%gist user/12345 %}` -> https://gist.github.com/user/12345
     - `{%slideshare user/my-awesome-presentation %}` -> Embedding removed
     - `{%speakerdeck foobar %}` -> Embedding removed
-- The use of `sequence` as code block language ([Why?](https://community.codimd.org/t/frequently-asked-questions/190))
```

Comment:
I would use the FAQ on the webpage instead.

Reply:
done